redesign/index.html

```diff
@@ -20,7 +20,7 @@
     <link rel="stylesheet" href="https://unpkg.com/tachyons@4.7.0/css/tachyons.min.css"/>
     <link rel="stylesheet" href="css/dwyl.css">
     <link rel="shortcut icon" href="/img/favicon.ico">
-    <title>dwyl</title>
+    <title>Agile development agency - dwyl</title>
```

Comment:
dwyl | App, Website and Software Development | London

Reply:
[...] Development & Training